### DISCUSSION TITLE
How to export the all schemas as one Schema file like schema.graphql

### DISCUSSION THREAD

**Discussion by khanakia** [99designs/gqlgen#3279] [Q&A]
I want to export the whole Schema Definition as one single file.

I know it can be done via introspection API. 

But is there any other way to use CLI ?

**Comment by UnAfraid** [comment]
If you're using Intellij based IDE like Goland for example there is a plugin that does that https://plugins.jetbrains.com/plugin/8097-graphql through introspection
Otherwise https://the-guild.dev/graphql/codegen can generate the schema and code for other languages like typescript, c#, java and others using introspection
You can find an example in https://github.com/99designs/gqlgen/tree/master/integration, most notably https://github.com/99designs/gqlgen/blob/master/integration/codegen.ts#L11-L13 and https://github.com/99designs/gqlgen/blob/master/integration/package.json#L8
